phpseclib/Crypt/Rijndael.php

```diff
@@ -755,7 +755,7 @@ function _subWord($word)
     {
         static $sbox;
         if (empty($sbox)) {
-            list(,,,, $sbox) = $this->_getTables();
+            list(, , , , $sbox) = $this->_getTables();
```

Comment:
Yeh, I think you're right. It's not "empty", and yeh, php's docs are a little ambiguous here.